packages/server/api/src/app/ai/ai-provider-proxy.ts

```diff
@@ -51,7 +53,8 @@ export const proxyController: FastifyPluginAsyncTypebox = async (
 
             const data = await parseResponseData(response, responseContentType)
 
-            await usageService(request.log).increaseProjectAndPlatformUsage({ projectId, incrementBy: 1, usageType: BillingUsageType.AI_TOKENS })
+            await platformUsageService.increasePlatformUsage(platformId, 1, UsageMetric.AI_CREDIT)
+            await projectUsageService(request.log).increaseProjectUsage(projectId, 1, UsageMetric.AI_CREDIT)
```

Comment:
What if you ever had `increaseProjectUsage` and forget to add `increasePlatformUsage` this sounds like we need to move them into one function, or move increasePlatformUsage into increaseProjectUsage